joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_1_to_arm_base
  pid: {p: 0.1, i: 0.1, d: 0.1}

joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_2_to_arm_1
  pid: {p: 2.0, i: 0.0, d: 0.0}

joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_to_arm_base
  pid: {p: 0.1, i: 0.1, d: 0.1}
  
  

